api/v1/views/index.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/python3
+"""
+Creating an Index
+"""
+
+
+from api.v1.views import app_views
+from flask import Flask, jsonify
+from models import storage
```

Comment:
Why import storage???

Reply:
Because is necesary to the execution of the file app.py to import all the modules and methods to run the file